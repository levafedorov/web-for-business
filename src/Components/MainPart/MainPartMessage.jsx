import React from 'react';




export default function MainScreen(props){
   
   
        
    
        return <div className="main-part-message">
                <h1>Welcome to</h1>
                <p>Yourcompanyname</p> 
               </div>
    
}