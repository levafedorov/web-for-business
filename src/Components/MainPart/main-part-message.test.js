import React from 'react';
import MainPartMessage from './MainPartMessage';
import {shallow, mount} from 'enzyme';







describe('CompactSlidingMenuTest component', () => {
    const setUp = (props) => shallow(<MainPartMessage {...props}/>);
    let component;
    let instance;
    beforeEach(() => {
        component = setUp();
        instance = component.instance();
    });

    it("Should components render", () => {
        component = setUp();
        expect(component).toMatchSnapshot();
    });

//     describe('Component methods', () => {
      
            
// });

});
