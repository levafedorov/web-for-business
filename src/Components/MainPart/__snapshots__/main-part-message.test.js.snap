// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompactSlidingMenuTest component Should components render 1`] = `
<div
  className="main-part-message"
>
  <h1>
    Welcome to
  </h1>
  <p>
    Yourcompanyname
  </p>
</div>
`;
